Snowflake.Data/Client/SnowflakeDbCommand.cs

```diff
@@ -25,6 +25,14 @@ public class SnowflakeDbCommand : DbCommand
 
         private SFLogger logger = SFLoggerFactory.GetLogger<SnowflakeDbCommand>();
 
+        public SnowflakeDbCommand()

```

Comment:
So the reason why you add parameterless constructor is because .net driver allows application to set Connection object after DbCommand is created, right?

Reply:
Yes, that's right.

Reply:
Specially in the added Factory class this is needed. It's late already here, I won't probably answer further questions today.